.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.analysis.typeCheckingMode": "off"
+}
```

Comment:
Can you please uncommit this file? (on your repo, just delete the file, add the change and commit)

Reply:
Just the file or the whole vscode repo?

Reply:
don't commit the content inside .vscode